common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -240,6 +271,14 @@
                     }
                 }
                 this.trigger('newPost:cancel');
+                parent.postMessage({event_name: 'discussion_post'}, '*');
+                url = DiscussionUtil.urlFor('discussion_celebration')
+                DiscussionUtil.safeAjax({
+                    url: url,
+                    type: 'POST',
+                    contentType: 'application/json',
+                    data: JSON.stringify({first_discussion: true}),
+                });
```

Comment:
This one will be removed. I'm using the cancel button for testing so I don't need to make a bunch of posts